src/core/Table/Table.tsx

```diff
@@ -640,38 +692,30 @@ export const Table = <
             className: cx('iui-table-body', {
               'iui-zebra-striping': styleType === 'zebra-rows',
             }),
+            style: { outline: 0 },
           })}
           ref={bodyRef}
           onScroll={() => {
             if (headerRef.current && bodyRef.current) {
               headerRef.current.scrollLeft = bodyRef.current.scrollLeft;
             }
           }}
+          tabIndex={-1}
         >
-          {data.length !== 0 &&
-            page.map((row: Row<T>) => {
-              prepareRow(row);
-              return (
-                <TableRowMemoized
-                  row={row}
-                  rowProps={rowProps}
-                  isLast={row.index === data.length - 1}
-                  onRowInViewport={onRowInViewportRef}
-                  onBottomReached={onBottomReachedRef}
-                  intersectionMargin={intersectionMargin}
-                  state={state}
-                  key={row.getRowProps().key}
-                  onClick={onRowClickHandler}
-                  subComponent={subComponent}
-                  isDisabled={!!isRowDisabled?.(row.original)}
-                  tableHasSubRows={hasAnySubRows}
-                  tableInstance={instance}
-                  expanderCell={expanderCell}
+          {data.length !== 0 && (
+            <>
+              {useVirtualization && (
+                <VirtualScroll
```

Comment:
I think the extra divs are breaking some styles. Notice the zebra rows are missing and some content is shifting
![rows-shift1](https://user-images.githubusercontent.com/9084735/151247196-f372bada-bedb-4c08-a5ed-37482c78a3f4.gif)

Reply:
Yes it does and I have created PR for that, but.. THere are other issues. Zebras are flickering, because odd/even index changes with virtualization. It needs js to be handled, so not sure..
Maybe separate PR?